wiki/en/Privacy-Statement.md

```diff
@@ -23,7 +23,7 @@ The website at [jamulus.io](https://jamulus.io) is served using GitHub Pages. We
 
 ### Use of Profile Information
 
-When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it. The Server does not otherwise store or record your Profile information and the Server operator has no access to it unless they are also connected as using a Client.
+When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it. The Server does not otherwise store or record your Profile information and the Server operator has no access to it unless they are also connected as using a Client or using JSON-RPC.
```

Comment:
This needs clarifying.  Does it mean using JSON-RPC to contact a client or to contact the server?

Reply:
Using JSON-RPC on the server to get a client list?

Reply:
You mean `jamulusserver/getClients`?
![image](https://user-images.githubusercontent.com/1549463/181996621-ec88c3fc-5263-449c-a209-915b998bc835.png)
That's what a Server operator can see when using the Server GUI.
```
"result": {
  "clients": [
    {
      "id": <channel id>,
      "address": <client ip:port>,
      "name": <name>,
      "jitterBufferSize": <jitter buffer size>,
      "channels": <channels>
    }, ...
  }
}
```
That's what `jamulusserver/getClients` returns - i.e. the same thing.  If that _is_ what you mean, it should mention _both_.

`jamulusserver/getServerProfile` gets the Server registration details, nothing about clients.

When you are running a Client that's connected to a Server, you can control the Client using RPC and - just as with the Client GUI, you can receive details of other clients on the server.  `jamulusclient/clientListReceived`  is the same as the Connect window.

In both cases, you need to be running Jamulus as either a Server or a Client and JSON-RPC doesn't expose _any_ information that isn't already exposed by that Server or Client _on the machine that Server or Client is running on and nowhere else_.

Reply:
```suggestion
When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it. The Server does not otherwise store or record your Profile information. The server operator might however use the Server-GUI or the JSON-RPC API to retrieve Profile information (including your IP-Address) during runtime.
```

Reply:
Better, though it's only the name field of the Client profile.

Reply:
Isn't the IP included too (at least for JSON-RPC)?

Reply:
The IP isn't present in the Client's Profile information.  It's derived from the UDP packet details, not the protocol.

Reply:
```suggestion
When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it. The Server does not otherwise store or record your Profile information. The server operator might however use the Server-GUI or the JSON-RPC API to see your Profile information while you are connected to a server.
```

Reply:
It's not the Client Profile information - except for the name.  I think it's misleading to say otherwise.  We don't show the server operator the skill level, instrument, location, language, etc.

Reply:
Ok. I agree that it can be misleading (it might change in future). Can you give an exact suggestion?

Reply:
Currently, an accurate statement would be 
```suggestion
When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it.  Note that, depending on the other client, you may not see them.  Also, the Server operator can see the name you set in My Profile and your IP address, using the Server either through the GUI or JSON-RPC.
```

Reply:
> Note that, depending on the other client, you may not see them.

I think what them is referring to is not clear enough. Everything else is OK, I think.

Reply:
@pljones could you please clarify what them is referring to: You might not see the other client at all or their information?